models.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "authors"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False, unique=True)
+    bio = Column(String(511), nullable=False)
+
+    books = relationship("DBBook", back_populates="author")
+
+
+class DBBook(Base):

```

Comment:
Rename it to `Book`. We don't use `DBrBook` to indicate it's a model; in FastAPI, we differentiate using `model.Author`, `schemas.Author`, etc.